src/tests/unit/tests/reports/components/formatted-date.test.tsx

```diff
@@ -1,23 +1,23 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import * as moment from 'moment';
+import { DateTime } from 'luxon';
 import { FormattedDate, FormattedDateProps } from 'reports/components/formatted-date';
 
 describe('FormattedDate', () => {
     describe('render', () => {
         test('end of last millennium', () => {
-            const date = moment.utc('1999-12-31T23:59:59').toDate();
-            testDate('en-us', date, '12/31/1999 11:59:59 PM UTC+00:00');
+            const date = DateTime.fromISO('1999-12-31T23:59:59', { zone: 'utc' }).toJSDate();
+            testDate('en-us', date, '12/31/1999, 11:59:59 PM UTC');
```

Comment:
(Existing issue) - is `expected` parameter to `testDate` redundant with the snapshot check?

Reply:
Yes, I agree. I think it's easier to see whether the test is testing the right thing with the inline version than the snapshot, so removing the snapshot